Xamarin.Forms.Platform.Android.UnitTests/Issues.cs

```diff
@@ -76,16 +77,16 @@ public async Task ToolbarItemWithReallyHighPriorityDoesntCrash()
 
 		[Test(Description = "No ObjectDisposedException should be thrown")]
 		[Issue(IssueTracker.Github, 5560, "[Android] Disposed EntryCell throws ObjectDisposed exception after updating an object that the EntryCell was previously bound to")]
-		public void EntryCellDisposed()
+		public async Task EntryCellDisposed()
```

Comment:
When running locally this test kept locking up my device every 2 or three runs so I just adjusted it slightly.

I made sure to test that it's still accurately testing the intended use case.